labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +84,8 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    error("Couldn't")
```

Comment:
As the assignment suggested, split was very useful to take each line and convert it into a list of words. Then you can use fold or map to convert each word to a length and get the longest.